### ISSUE TITLE
[Programmatic Access - Bot Framework - Keyboard controls]: Ensures aria-hidden elements are not focusable nor contain focusable elements (.webchat__keyboard-help).

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4714]
### **User Experience:**
Keyboard users and people who rely on assistive technologies and users get affected if the focus lands on the hidden elements while navigating. Users might consider it as an interactive control and try to access them but visually the controls are not visible and hence get confused.

### **Note:** 
User credentials should NOT be included in the bug.
### **Pre-Requisite:**

- Open environment URL https://compulim.github.io/webchat-loader/.
- Select "Cdn.botframework.com/latest" list item under version combo box.
- Select "Mock Bot" link and select "Open web chat in new window" link.

### **Repro Steps:**

1. Open "Web chat" link and navigate to "Webchat" popup.
2. Navigate to the "Keyboard Controls" pane. Turn on AI-fast pass.
3. Verify whether the hidden elements have focus or not.

### **Actual Result:**
The aria-hidden elements are focusable nor contain focusable elements (.webchat__keyboard-help).

### **Expected Result:**
Ensures aria-hidden elements are not focusable nor contain focusable elements (.webchat__keyboard-help). 

### **Issue:** 
Ensures aria-hidden elements are not focusable nor contain focusable elements (aria-hidden-focus - https://accessibilityinsights.io/info-examples/web/aria-hidden-focus)

### **Target application:** 
Web Chat - https://compulim.github.io/webchat-loader/webchat.html?v=https%3A%2F%2Fcdn.botframework.com%2Fbotframework-webchat%2Flatest%2F&p=ws&sr=westus2&uid=dl_edaa29c6e1e78d84dc6aecfaeb7c83c9&cid=DbsRxl7sUlN2Yqll5suvU8-us&dt=eyJhbGciOiJSUzI1NiIsImtpZCI6ImFkRnJFUkU5bFFUX011c3hlajd2NlRscVU5WSIsIng1dCI6ImFkRnJFUkU5bFFUX011c3hlajd2NlRscVU5WSIsInR5cCI6IkpXVCJ9.eyJib3QiOiJ3ZWJjaGF0LW1vY2tib3QiLCJzaXRlIjoiV1lESXlLd0FaQ3ciLCJjb252IjoiRGJzUnhsN3NVbE4yWXFsbDVzdXZVOC11cyIsInVzZXIiOiJkbF9lZGFhMjljNmUxZTc4ZDg0ZGM2YWVjZmFlYjdjODNjOSIsIm9yaWdpbiI6WyJodHRwczovL3dlYmNoYXQtcGxheWdyb3VuZC5henVyZXdlYnNpdGVzLm5ldCIsImh0dHA6Ly9sb2NhbGhvc3Q6MzAwMCIsImh0dHBzOi8vbWljcm9zb2Z0LmdpdGh1Yi5pbyIsImh0dHBzOi8vY29tcHVsaW0uZ2l0aHViLmlvIiwiaHR0cHM6Ly9iZnh3ZWJjaGF0ZnVsbGJ1bmRsZS5henVyZXdlYnNpdGVzLm5ldCIsImh0dHA6Ly9sb2NhbGhvc3Q6NTA4MCJdLCJuYmYiOjE2ODIwNTg0NzQsImV4cCI6MTY4MjA2MjA3NCwiaXNzIjoiaHR0cHM6Ly9kaXJlY3RsaW5lLmJvdGZyYW1ld29yay5jb20vIiwiYXVkIjoiaHR0cHM6Ly9kaXJlY3RsaW5lLmJvdGZyYW1ld29yay5jb20vIn0.p6x1j8c-SMGSssE0Ckb58Ikq2S2ax8iZ-hhT24xyAZN5UqN7s0lNZjoaukIueRrh9ZuqztD652JzvKIsftQ9uwRdmFZPognKrR2y9e33Z65YgH7qVpi-51d8P0SG-16OEn16wIn_yr4F125oJDM9qIigHmamuQ7nAlc2b_MbbXZ4x6SQFGurbFgY4QROGt7VyTr9J2weDgvEFUjNyWBPK-2w58TBwac-d6ORMH8w4IcaJv7No8awJ7fMaN7UJRAYY_mTwMy1spNZqDbUEUWPBT6lS3Yt62fNXJLwXHEl8U_dORtfn8o-YEJnoNDqP1RneXmuCQ6rjY8mg8o8yhf7sg&st=eyJhbGciOiJFUzI1NiIsImtpZCI6ImtleTEiLCJ0eXAiOiJKV1QifQ.eyJyZWdpb24iOiJ3ZXN0dXMyIiwic3Vic2NyaXB0aW9uLWlkIjoiNTZmM2YwMWVmODFkNDA2ZDllZWFhMGU1OTI2NTdlN2MiLCJwcm9kdWN0LWlkIjoiU3BlZWNoU2VydmljZXMuUzAiLCJjb2duaXRpdmUtc2VydmljZXMtZW5kcG9pbnQiOiJodHRwczovL2FwaS5jb2duaXRpdmUubWljcm9zb2Z0LmNvbS9pbnRlcm5hbC92MS4wLyIsImF6dXJlLXJlc291cmNlLWlkIjoiL3N1YnNjcmlwdGlvbnMvMzVmYTc0NjEtZDQ4YS00MDUzLTlhYTItNTZjMDE0NDk1ZTg5L3Jlc291cmNlR3JvdXBzL1dlYkNoYXRQcm9kdWN0aW9uL3Byb3ZpZGVycy9NaWNyb3NvZnQuQ29nbml0aXZlU2VydmljZXMvYWNjb3VudHMvd2ViY2hhdC1tb2NrYm90LWN1c3RvbXNwZWVjaCIsInNjb3BlIjoic3BlZWNoc2VydmljZXMiLCJhdWQiOiJ1cm46bXMuc3BlZWNoc2VydmljZXMud2VzdHVzMiIsImV4cCI6MTY4MjA1OTA3NSwiaXNzIjoidXJuOm1zLmNvZ25pdGl2ZXNlcnZpY2VzIn0.LWtFmD1QKmf_OwaraDw3GoVY1feAdvvsY1PGfSwIECewNf6-PFySWfn6ARFE1ZrNVGcpm2-_9eJ5UzcBoSob5g.

### **Element path:** 
.webchat__keyboard-help

### **Snippet:** 
<div
aria-hidden="true"
aria-labelledby="webchat__keyboard-help__header--zs150"
class="webchat__keyboard-help webchat--css-lcohw-at18kd"
role="dialog">

### **How to fix:**
Fix all of the following:

-  Focusable content should be disabled or be removed from the DOM

### **MAS Reference:**
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)
### **Reference Links:**

- [Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-poyad%40microsoft.com%7cc11478591b5a474f6d9d08d7199726c0%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637006009300301864&sdata=ZTWYamT1yE7yko5QU35ZJLf/5ZydtDeGAKrQGUiugJs%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [Link](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDV8MDF8fDNkZjhhMWU0NTA0YTQwM2MyNGRhMDhkYWY1NTkwMDNlfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgwOTIwNTk4MzEzMTM2MjZ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVFkzTXpZd09URTRNall3TXpzeE5qY3pOakE1TVRneU5qQXpPekU1T2pjMllUWXlNRE5pTnpZeFpqUmlObVE0T1dKbE0yRm1aVFZpTjJZd1pHSmhRSFJvY21WaFpDNTJNZz09fDBiZDAxMWU1M2M0YjQ0NGYyNGRhMDhkYWY1NTkwMDNlfGI0NDRlYzhjODQ4YzRiODI4Mzk1NmRjNDdlZjQ3NDky&sdata=dlJVM3lpMWQrRmtpc09HVTI0REh4NlVrU00yMG5xd0JlUFdBOHZuaHNOVT0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-pgopinadhu%40microsoft.com&OR=Teams-HL&CT=1673616420039&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yMzAxMDUwOTQwMCIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D). “HCL Staff should not log any third-party external bugs. Should be routed to edad team”.
- Please reach out to [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process related queries.

**Comment by pgopinadhu** [comment]
HCL; Accessibility; Benchmark; A11yMAS; A11ySev2; MAS4.1.2; A11yAuto; K4W; Win11-Edge(Chromium); Feature-HCL-Botframeworkwebchat-Apr23

**Comment by pgopinadhu** [comment]
<img width="1788" alt="AI-MAS4 1 2-botframework-keyboard-hidden" src="https://user-images.githubusercontent.com/120019572/233827154-53125268-f4cb-4426-a1f7-8eb4a1cc2938.png">

**Comment by compulim** [comment]
This is fixed in #4659 and released as 4.15.8.

In 4.15.8, the `aria-hidden` is removed.
<img width="334" alt="image" src="https://github.com/microsoft/BotFramework-WebChat/assets/1622400/411b461c-4e5b-4633-9715-384010be0356">

In 4.15.7, the `aria-hidden` is present.
<img width="333" alt="image" src="https://github.com/microsoft/BotFramework-WebChat/assets/1622400/345c6c46-5b68-44a6-885a-071df377fb35">

**Comment by pgopinadhu** [comment]
As verified the issue on https://compulim.github.io/webchat-loader/ under Windows 11 Enterprise 22H2 (OS build: 22621.1702), Edge Version 115.0.1851.0 (Official build) dev (64-bit) and observed that the issue is fixed.
